### PR TITLE
Added calibration script

### PR DESCRIPTION
**Pull request by Bir9**

### CODE REVIEW

File: calibration.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3   
+import time 
+from robot import
```
Comment (Bir9) [inline calibration.py:3]: add Robot to the end of this line

File: calibration.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3   
+import time 
+from robot import 
+import main
+from pybricks.parameters import Button
+def calibration(ev3, left_cs, right_cs, right_wheel, left_wheel): #add any extra sensor/motors, I can not as I would get an error
+    button_pressed = False
+    counter = 0
```
Comment (Bir9) [inline calibration.py:8]: remove counter and delay variables

File: calibration.py (hunk 1 above)
Comment (bashir2) [reply calibration.py:3]: @Bir9, can you please fix this line and use an instance of the robot in the `calibration` function?

File: calibration.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3   
+import time 
+from robot import 
+import main
```
Comment (bashir2) [inline calibration.py:4]: Why do we need this import? This script should work independent of `main`.

**Review by bashir2 — COMMENTED** [review]
Thanks @Bir9 added some suggestions/questions.

File: calibration.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        ev3.screen.draw_text(0, 60, left_wheel_string)
+        ev3.screen.draw_text(0, 90, right_wheel_string)
+        print(ev3.buttons.pressed())
+        time.sleep(0.2)
+        ev3.screen.clear()
+        if(ev3.buttons.pressed() == [Button.CENTER]):
+            button_pressed = True
+        else:
+            button_pressed = False
+    str_value = str(left_cs.reflection())
```
Comment (bashir2) [inline calibration.py:26]: This should be the maximum of `left_cs` and `right_cs` plus a positive number, e.g., `2` to have some margin of error.

**Comment by bashir2** [comment]
This is superseded by #17; closing ...

**Pull request closed without merging**